iflipb.el

```diff
@@ -179,6 +179,26 @@ after editing a buffer will act as if the current buffer was not
 visited; it will stay in its original place in the buffer list."
   :group 'iflipb)
 
+(defface iflipb-buffer-face
+  '((t (:foreground "default")))
+  "Face used for a non-current buffer name."
+  :group 'iflipb)
+
+(defface iflipb-current-buffer-face
+  '((t (:foreground "white")))
+  "Face used for the current buffer name."
+  :group 'iflipb)
+
+(defcustom iflipb-buffer-template
+  "%s"
+  "The template string that will be applied to a non-current buffer name. Use string `%s` to refer to the buffer name."
```

Comment:
The Emacs doc string convention is to wrap lines at 65 characters, so I suggest doing that (try `M-q` inside the doc string!). Another convention is to use `` `...' `` instead of `` `...` ``.